### ISSUE TITLE
[FEATURE] Caching of images and other files in website

### ISSUE THREAD

**Issue by SaptarshiSarkar12** [SaptarshiSarkar12/Drifty#64]
### Describe the solution you'd like.

Caching the images and other such files for the website.

### Application Version

v1.1.0

### Additional information

If anyone wants to work on this issue, please comment below and I shall assign you with the task.

**Comment by c99SRS** [comment]
HI, @SaptarshiSarkar12 can you please assign this to me? We can discuss this before implementation

**Comment by SaptarshiSarkar12** [comment]
@c99SRS How much work is done?

**Comment by c99SRS** [comment]
Hi @SaptarshiSarkar12 I am working on it. Need some discussion with you as well.

**Comment by SaptarshiSarkar12** [comment]
> Hi @SaptarshiSarkar12 I am working on it. Need some discussion with you as well.

Ok @c99SRS, we shall discuss on this. Do you want to discuss here or on any other platform like discord, twitter etc.?

**Comment by c99SRS** [comment]
Yes, discord is fine. please share your discord  details

**Comment by c99SRS** [comment]
HI @SaptarshiSarkar12 was busy a little bit. unable to find you. can you pls add me. mine is **SmrutiRanjan#8027**

**Comment by SaptarshiSarkar12** [comment]
> 

Ok, let me check.

**Comment by SaptarshiSarkar12** [comment]
> HI @SaptarshiSarkar12 was busy a little bit. unable to find you. can you pls add me. mine is **SmrutiRanjan#8027**

@c99SRS I have sent a friend request, accept that and send a message to start the discussion.

**Comment by SaptarshiSarkar12** [comment]
@c99SRS How much work is done?

**Comment by SaptarshiSarkar12** [comment]
> @c99SRS How much work is done?

Please respond @c99SRS .

**Comment by SaptarshiSarkar12** [comment]
> > @c99SRS How much work is done?
> 
> Please respond @c99SRS .

@c99SRS

**Comment by SaptarshiSarkar12** [comment]
@c99SRS Are you willing to work on this issue?

**Comment by SaptarshiSarkar12** [comment]
Unassigning @c99SRS from the issue because of inactivity. This issue is now open for contributors.

**Comment by sreevyshnavi02** [comment]
Hello! 
Can I use a free, open source CDN(Content Delivery Network) like **jsDelivr** to cache the files?

Since the website is hosted on Github pages, I believe we cannot use a language like PHP for caching the data.

**Comment by SaptarshiSarkar12** [comment]
> Hello! Can I use a free, open source CDN(Content Delivery Network) like **jsDelivr** to cache the files?
> 
> Since the website is hosted on Github pages, I believe we cannot use a language like PHP for caching the data.

Yes, you can use **jsDelivr** . The website is going to be changed to a React Application via #196 , so you need to wait until a Pull Request for the new website is created, reviewed and merged. I am assigning you with the task.
